workflow/repo_data/config/config.default.yaml

```diff
@@ -18,6 +18,13 @@ scenario:
   sector: "" # G
   planning_horizons: [2030, 2040, 2050]
 
+# docs :
+model_topology:
+  transmission_network: 'tamu' # [reeds, tamu]
+  include: # mixed zone types not supported
+    reeds_zone: ['p8']
+    # reeds_state: ['CO']
```

Comment:
Im not sure if I am missunderstanding, but I am having trouble getting these config options to work properly. If I change the transmission network to reeds, and set the topological_boundaries to either `reeds_zone` or `state`, these both fail. I run into the `n_clusters` assertion, and make sure I respect the clustering requirements.

Reply:
Also the `ValueError` in `build_bus_regions` says something different 

```bash
ValueError: zonal_aggregation must be either balancing_area, country, reeds_id, or state
```

Reply:
Topological boundaries shouldn't change from reeds_zone. I was considering getting rid of that config but whated to check with you. Okay, I'm able to run with  `topological_boundaries: reeds_zone ` so lets review this. The n_clusters assertion is required if you use `transmission_network: reeds`

Reply:
You have to set `transmission_network` to `tamu` if you want to use the `include` functions? It wont work if its `reeds`?

Reply:
resolving since we realized we need an assertion that the inclusion zones exist within the interconnect